### ISSUE TITLE
loss nan

### ISSUE THREAD

**Issue by vcvishal** [experiencor/keras-yolo2#422]
when i use it on my dataset it shows
Current Recall  [0]
Average Recall  [0]
Loss XY         [nan]
Loss WH         [nan]
Loss Conf       [nan]
Loss Class      [nan]
Total Loss      [nan]
Current Recall  [0]
Average Recall  [0]
Loss XY         [nan]
Loss WH         [nan]
Loss Conf       [nan]
Loss Class      [nan]
Total Loss      [nan]
Current Recall  [0]
Average Recall  [0]
 - 7s - loss: nan - val_loss: nan

please guide me
thank you
